R/tax_certainty.R

```diff
@@ -0,0 +1,33 @@
+#' Check if a taxon is certain or uncertain.
+#'
+#' Check whether a given taxon in a vector is certain or uncertain.
+#' Binomial names (genus and species) are considered "certain",
+#' whereas names containing only the genus,
+#' or genus and not determined species is considered "uncertain".
+#'
+#' @param taxon_names Character vector containing the species names to be
+#' checked.
+#' It allows taxon names separated by underscore or space.
+#' @return Numeric vector classifying each species as certain (1) or uncertain
+#' (0).
+#' @examples
```

Comment:
```suggestion
#' @section Developer(s):
#'     Bruna Farina
#' @section Reviewer(s):
#'     Lewis A. Jones
#' @examples
```
We normally try to keep track of this in the documentation to acknowledge one another and serve as a point of contact for any future updates.